fbbt/src/ontologies/fbbt-edit.obo

```diff
@@ -63575,6 +63576,95 @@ intersection_of: FBbt:00005106 ! neuron
+is_a: FBbt:00007701 ! longitudinal subdivision of central nervous system
+relationship: part_of FBbt:00001102 ! ventral nerve cord
+property_value: created_by "dos" xsd:string
+creation_date: 2018-01-22T14:05:07Z
+
+[Term]
+id: FBbt:00007706
+name: ventromedial subdivision of ventral nerve cord
+def: "A ventro-medially located longitudinal subdivision of the larval ventral nerve cord spanning the gnathal, thoracic and abdominal segments to abdominal segment 8.  A virtual axial plane intersecting the CITv and the VMT (`Ventro Medial Tract` [FBbt_00001646]) forms the ventral boundary of this domain, the lateral boundary ventral end of the vertical medial and posterior medial bundles. (CITv encompasses `CI2`[FBbt_00005906] and `CI3`[FBbt_00005907])." [FB:dos] {comment="PMID:28730682"}
+is_a: FBbt:00007701 ! longitudinal subdivision of central nervous system
+relationship: part_of FBbt:00001102 ! ventral nerve cord
+property_value: created_by "dos" xsd:string
+creation_date: 2018-01-22T14:06:02Z
+
+[Term]
+id: FBbt:00007707
+name: dorsolateral subdivision of ventral nerve cord
+def: "A dorso-laterally located longitudinal subdivision of the larval ventral nerve cord spanning the gnathal, thoracic and abdominal segments to abdominal segment 8. An axial plane just dorsal to the intermediate commissures and CITd (centro intermediate tract dorsal, `CI1` [FBbt_00005905]) specifies the ventral boundary of this domain.  It is separated from the dorsomedial domains by a sagittal plane intersecting the CITd fascicle." [FB:dos] {comment="PMID:28730682"}
+is_a: FBbt:00007701 ! longitudinal subdivision of central nervous system
+relationship: part_of FBbt:00001102 ! ventral nerve cord
+property_value: created_by "dos" xsd:string
+creation_date: 2018-01-22T14:06:37Z
+
+[Term]
+id: FBbt:00007708
+name: dorsomedial subdivision of ventral nerve cord
+def: "A dorso-medially located longitudinal subdivision of the larval ventral nerve cord spanning the gnathal, thoracic and abdominal segments to abdominal segment 8. An axial plane just dorsal to the intermediate commissures and CITd (centro intermediate tract dorsal, `CI1` [FBbt_00005905]) specifies the ventral boundary of this domain.  It is separated from the dorsolateral domains by a sagittal plane intersecting the CITd fascicle." [FB:dos] {comment="PMID:28730682"}
+is_a: FBbt:00007701 ! longitudinal subdivision of central nervous system
+relationship: part_of FBbt:00001102 ! ventral nerve cord
+property_value: created_by "dos" xsd:string
+creation_date: 2018-01-22T14:07:31Z
+
+[Term]
```

Comment:
below OK but missing note about CITv in others:
(CITv encompasses `CI2`[FBbt_00005906] and `CI3`[FBbt_00005907]).